<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/icons/ru.png</normaloff>:/icons/ru.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralWidgetLout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionBrowseNetworkDevices"/>
   <addaction name="actIpAddrConfig"/>
   <addaction name="actionSettingsDialog"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDaq">
    <property name="title">
     <string>Data Ac&amp;quisition</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionDraw_Plot"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>De&amp;vices</string>
    </property>
    <addaction name="actionRefreshDevList"/>
    <addaction name="actIpAddrConfig"/>
    <addaction name="actionBrowseNetworkDevices"/>
    <addaction name="actionDevCfg"/>
    <addaction name="actionDevIfaceSettings"/>
    <addaction name="actionDevUpdateFirmware"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettingsDialog"/>
    <addaction name="actionSaveDataBlock"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Wi&amp;ndows</string>
    </property>
    <addaction name="actionShowDevsPanel"/>
    <addaction name="actionShowSigParamsPanel"/>
    <addaction name="actionShowLogPanel"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionLangSystem"/>
    <addaction name="actionLangRussian"/>
    <addaction name="actionLangEnglish"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevices"/>
   <addaction name="menuDaq"/>
   <addaction name="menuWindows"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QDockWidget" name="devicePanel">
   <property name="windowTitle">
    <string>Devices panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="refreshDevButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="devicesTree">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDevCfg">
   <property name="text">
    <string>&amp;Device Configuration...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start data acquisition/generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>St&amp;op</string>
   </property>
   <property name="toolTip">
    <string>Stop data acquisition/generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionRefreshDevList">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh device list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionSettingsDialog">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Application Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShowLogPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Log</string>
   </property>
   <property name="toolTip">
    <string>Show log panel</string>
   </property>
  </action>
  <action name="actionShowSigParamsPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Signal Parameters</string>
   </property>
   <property name="toolTip">
    <string>Show panel with signal parameters</string>
   </property>
  </action>
  <action name="actionShowDevsPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Devices Panel</string>
   </property>
   <property name="toolTip">
    <string>Show panel with devices tree</string>
   </property>
  </action>
  <action name="actIpAddrConfig">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/ipaddr.png</normaloff>:/icons/ipaddr.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure Connection IP Addresses...</string>
   </property>
   <property name="toolTip">
    <string>Configure connection IP addresses...</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Sa&amp;ve Screenshot</string>
   </property>
  </action>
  <action name="actionSaveDataBlock">
   <property name="text">
    <string>&amp;Saving Data Locationâ€¦</string>
   </property>
   <property name="toolTip">
    <string>Set the location to save the measured data</string>
   </property>
  </action>
  <action name="actionDevIfaceSettings">
   <property name="text">
    <string>&amp;Modify Interface Configuration...</string>
   </property>
  </action>
  <action name="actionDevUpdateFirmware">
   <property name="text">
    <string>&amp;Update Firmware...</string>
   </property>
  </action>
  <action name="actionLangSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;System</string>
   </property>
   <property name="toolTip">
    <string>Set system language</string>
   </property>
  </action>
  <action name="actionLangRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/ru.png</normaloff>:/icons/ru.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Russian</string>
   </property>
   <property name="toolTip">
    <string>Set Russian language</string>
   </property>
  </action>
  <action name="actionLangEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/en.png</normaloff>:/icons/en.png</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
   <property name="toolTip">
    <string>Set English language</string>
   </property>
  </action>
  <action name="actionBrowseNetworkDevices">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/netbrowse.png</normaloff>:/icons/netbrowse.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Browse Local Network Devices...</string>
   </property>
   <property name="toolTip">
    <string>Browse Local Network Devices...</string>
   </property>
  </action>
  <action name="actionAsyncIO">
   <property name="text">
    <string>Asynchronous Input/Output...</string>
   </property>
  </action>
  <action name="actionDraw_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Plot</string>
   </property>
   <property name="statusTip">
    <string>When unticked, reduces the CPU load many times</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>stopCurConnection()</slot>
  <slot>startCurConnection()</slot>
  <slot>resetCurModule()</slot>
  <slot>addLtrdConnectionDialog()</slot>
  <slot>remLtrdConnection()</slot>
  <slot>LtrdAutoconChanged()</slot>
  <slot>ltrdConnectionSettingsDialog()</slot>
  <slot>showHideMainWindow()</slot>
 </slots>
</ui>
